lib/gui/app/modules/drive-scanner.js

```diff
@@ -23,7 +23,7 @@ const permissions = require('../../../shared/permissions')
 const scanner = SDK.createScanner({
   blockdevice: {
     get includeSystemDrives () {
-      return settings.get('unsafeMode') && !process.env.ETCHER_HIDE_UNSAFE_MODE
+      return settings.get('unsafeMode') && !settings.get('disableUnsafeMode')
```

Comment:
I think `hideUnsafeMode` is a better description given that it's disabled by default, and we're actually hiding the ability to toggle unsafe mode while still keeping it disabled

Reply:
IMHO if this was indeed a `hideUnsafeMode` setting, then that'd imply that this `includeSystemDrives` function would simply be `return settings.get('unsafeMode')` (i.e. unsafeMode could remain enabled, even if the option to toggle it was hidden).
However as that's not the case, I think `disableUnsafeMode` is perhaps the best name? :man_shrugging:

Reply:
That's true, but our intent with this option (and these options generally) is to create an environment rather than just an option you toggle every now and then as a user, as such it's safe to assume it will remain disabled.

Reply:
Yeah, I think sticking with "disable" is more appropriate still